packages/devtools_app/lib/src/framework/app_error_handling.dart

```diff
@@ -22,24 +22,26 @@ import '../shared/config_specific/logger/logger.dart';
 void setupErrorHandling(Future Function() appStartCallback) {
   // First, run all our code in a new zone.
   return runZonedGuarded(
-      // ignore: avoid-passing-async-when-sync-expected this ignore should be fixed.
-      () async {
-    WidgetsFlutterBinding.ensureInitialized();
-
-    final FlutterExceptionHandler? oldHandler = FlutterError.onError;
-
-    FlutterError.onError = (FlutterErrorDetails details) {
-      _reportError(details.exception, details.stack ?? StackTrace.empty);
-
-      if (oldHandler != null) {
-        oldHandler(details);
-      }
-    };
-
-    return appStartCallback();
-  }, (Object error, StackTrace stack) {
-    _reportError(error, stack);
-  });
```

Comment:
Example of fixes that I think match what @kenzieschmoll would suggest in a code review.